api_v3/services/PartnerService.php

```diff
@@ -139,7 +139,8 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 		{
 			KalturaLog::CRIT($ex);
 			// this assumes the partner name is unique - TODO - remove key from DB !
-			throw new KalturaAPIException( APIErrors::PARTNER_REGISTRATION_ERROR, 'Unknown error');
+			$exceptionMessage = (($pos = strpos($ex->getMessage(), 'KalturaPartner::')) !== FALSE) ? str_replace('KalturaPartner::', '', $ex->getMessage()) : $ex->getMessage();
```

Comment:
The check is not really needed,
If you just remove 'KalturaPartner::' the results will be the same.
str_replace('KalturaPartner::', '',"TestKalturaPartner::123");
will yeild the same result as 
str_replace('KalturaPartner::', '',"Test123");